components/homme/test_execs/share_kokkos_ut/CMakeLists.txt

```diff
@@ -82,6 +82,8 @@ SET (BOUNDARY_EXCHANGE_UT_F90_SRCS
   ${UTILS_SHARE_DIR}/shr_kind_mod.F90
   ${UTILS_SHARE_DIR}/shr_mpi_mod.F90
   ${UTILS_SHARE_DIR}/shr_sys_mod.F90
+  ${UTILS_TIMING_DIR}/perf_mod.F90
+  ${UTILS_TIMING_DIR}/perf_utils.F90
```

Comment:
These tests run in nightlies, just out of curiosity, is this somehow an issue on the testbed you use?

Reply:
I think the only tests being run are on cori-knl. So this was an issue found using the environment on cori-gpu.

Reply:
Unit tests run on all machines that run hommebfb suite, like, for example, here
```
      Start 79: eos_ut_test
79/93 Test #79: eos_ut_test ....................................................................   Passed    2.93 sec
      Start 80: elem_ops_ut_test
80/93 Test #80: elem_ops_ut_test ...............................................................   Passed    3.18 sec
      Start 81: hv_ut_test
81/93 Test #81: hv_ut_test .....................................................................   Passed    3.41 sec
...
```
So, I am curious why they run without these changes.